### PR TITLE
chore: blog responsiveness

### PR DESCRIPTION
**Pull request by paulatulis**
Done: Input no longer cut off

To Do: make blog cards look more polished based on feedback from Phil

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit a7010b8046b2dae41002412444ca92781ffc8dbc

https://deploy-preview-1506--stoplightio.netlify.app

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 4f35fa038a918023346b1a378eebafe837d2e26e

https://deploy-preview-1506--stoplightio.netlify.app

**Comment by ksatx** [comment]
Still cutting off a tiny bit for me.  What if I made the form helper text "Get the newsletter" and the button just say "Subscribe"?


15448ba9-be5a-456a-a12b-e1e10f87fa27

File: netlify/lists/blog.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ hero:
           type: email
           placeholder: Your email...
```
Comment (ksatx) [inline netlify/lists/blog.yaml:17]:
```suggestion
          placeholder: Get The Newsletter
```

**Comment by paulatulis** [comment]
@ksatx can we try one more thing before changing the text? I think it's best to keep the helper text inside the input as email if we can. I tried updating the font size and div width. When I toggle to small screen it looks like this: 
Screen Shot 2020-06-30 at 2 38 51 PM

What's it doing for you now?

**Comment by ksatx** [comment]
@paulatulis Yeah, it is nice to have the 'your email' helper text. What about a media query that changes the button to just "Subscribe" on mobile, or moved the submit below the form? Maybe that's overthinking it.

It's still a little bit wonky on my phone.

Screenshot_20200630-145714

**Comment by paulatulis** [comment]
How does it look for you now? I updated it to just stack on small screens.
Screen Shot 2020-06-30 at 3 32 54 PM

**Comment by ksatx** [comment]
It looks great, yay!!!

**Comment by paulatulis** [comment]
I will merge this and can open a new PR when we have more feedback about other aesthetics

**Pull request merged**